src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static class EnumTests
+    {
+        private static readonly string s_jsonStringEnum =
+                @"{" +
+                @"""MyEnum"" : ""Two""" +
+                @"}";
```

Comment:
**Out of curiosity**, why produce input strings like this rather than something like one of these:
```C#
private static readonly string s_jsonIntEnum = @"{""MyEnum"" : 2}";
private static readonly string s_jsonIntEnum =
@"
{
   ""MyEnum"" : 2
}";
private static readonly string s_jsonIntEnum = "{\n\"MyEnum\" : 2\n}";
```

Reply:
For simple cases, yes it could be a single string. But mostly this is consistent with the more complicated scenarios where having the braces and array brackets on separate lines makes it more readable and easier to copy-paste.